src/FieldWarning/Assets/Model/Game/MatchSession.cs

```diff
@@ -92,6 +94,7 @@ public void Awake()
             PathData = new PathfinderData(GameObject.Find("Terrain").GetComponent<Terrain>());
             Factory = new UnitFactory();
             Settings = new Settings();
+            
```

Comment:
Try to avoid committing files if you only made (unintended) whitespace changes. It is not a big error or anything but it's nice to take a look at the files before and after making the commit to make sure the contents are what you expect them to be.